content/documentation/pages/5-feature-guides/2-streams/1-deployment-properties.md

```diff
@@ -52,3 +63,12 @@ The preceding example would be as follows in SCDF shell:
 stream deploy --name ticktock --properties "app.time.trigger.initial-delay=1,deployer.*.cpu=1,deployer.*.local.shutdown-timeout=60,deployer.*.memory=512,deployer.log.count=2,deployer.log.local.delete-files-on-exit=false,deployer.time.disk=512,spring.cloud.dataflow.skipper.platformName=local-debug"
 Deployment request has been sent for stream 'ticktock'
 ```
+
+## Platform Specific Deployer Properties
+
+### Cloud Foundry Deployer Properties
+
+The Cloud Foundry Deployer maps task and application properties to environment variable `SPRING_APPLICATION_JSON` by default. This is defined in the generated Cloud Foundry application manifest.
```

Comment:
Change the first sentence to: The Cloud Foundry Deployer maps task and application properties to an environment variable (by default, `SPRING_APPLICATION_JSON`).

Reply:
This does not convey the same meaning to me.  `SPRING_APPLICATION_JSON` is a special boot variable. The name is not a default setting.  The alternative configuration maps each property to a separate env variable.

Reply:
But a good catch anyway. I meant to remove this section.